### PR TITLE
Add option for showing ETA, fix bug parsing distancetoturn, change translate function

### PR DESCRIPTION
**Pull request by Niklas04**
Add option for showing ETA
Fix bug parsing distancetoturn by extras
Add query for notification group by parsing by extras
Change translation-function to work with all languages (strings.xml)
Showing distance to turn "0"m when GoogleMaps shwoing "0"m

### CODE REVIEW

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java [hunk 1]
```diff
@@ -437,25 +437,26 @@ private static eUnits get_eUnits(String unit) {
...
-            case "小時":
-            case "時":
-            case "시간":
-                return "h";
-            default:
-                return null;
-        }
+    // Translates the units (distance and time) from local language and charset in common values
+    private String translate(String local_language_string) {
+        if(local_language_string == getString(R.string.km))
```
Comment (skyforcetw) [inline GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: Nice job.
I want to change the string judgement by R.string, but switch/case cannot support dynamic values.
I forgot if /else can do this.......

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java [hunk 2]
```diff
@@ -120,8 +120,7 @@ private static void logi(String msg) {
         @Override
         public void onReceive(Context context, Intent intent) {
 
-            int showETAValue = intent.getIntExtra(Integer.toString(R.id.switchShowETA), 0);
-            showETA = 0 == showETAValue ? showETA : 2 == showETAValue;
+            showETA = intent.getBooleanExtra(Integer.toString(R.id.switchShowETA), false);
```
Comment (skyforcetw) [inline GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: I think there will be a little dangerous.
When I switch showETA in MainAcitivity, it'ok.
When I switch other option in MainAcitivity (but there is no other option right now), not switchShowETA extra in intent, showETA will be turn off.

So I change from send bool to send integer, for two or more options in MainAcitivity.

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java (hunk 1 above)
Comment (Niklas04) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: Thanks
Yes had the same problem, but then found that solution

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java (hunk 2 above)
Comment (Niklas04) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: Hm, got what you mean.

Just a thought, but may it is possible too, to use the follwing code:
showETA = intent.getBooleanExtra(Integer.toString(R.id.switchShowETA), showETA);

Comment (skyforcetw) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: It should be ok!

**Pull request merged**